cluster/src/main/java/com/hartwig/pipeline/resource/RefGenome38ResourceFiles.java

```diff
@@ -135,16 +135,6 @@ public String svBreakpointPon() {
         return formPath(GRIDSS, "sv_pon.38.bedpe.gz");
     }
 
-    @Override
-    public String fragileSites() {
-        return formPath(LINX, "fragile_sites_hmf.38.csv");
-    }
-
-    @Override
-    public String lineElements() {
-        return formPath(LINX, "line_elements.38.csv");
-    }
-
```

Comment:
Is this just opportunistic cleanup or is it related to one of the other tool version bumps somehow (just curious)?

Reply:
Linx no longer loads these resource files